src/styles/view-switcher.scss

```diff
@@ -0,0 +1,61 @@
+.view-switcher {
+  display: flex;
+  justify-content: center;
+
+  // These are the exact margins required to vertically align the top of the
+  // view switcher with the top of the Hide Highlights button to its left,
+  // and the top of the first annotation card below the view switcher with the
+  // top of the New Page Note button to its left.
+  margin-top: 1px;
+  margin-bottom: 6px;
+}
+
+.view-switcher__tab {
+  @include smallshadow
+
+  height: 30px; // Same height as used for Hide Highlights and New Page Note
+                // buttons to left of view switcher.
+  padding-left: 12px;
+  padding-right: 12px;
+
+  background: $white;
+  transition: background-color .1s linear;
+
+  border: 1px solid $gray-lighter;
+
+  touch-action: manipulation;
+  cursor: pointer;
+  user-select: none;
+
+  -webkit-appearance: button;
```

Comment:
What effects does `-webkit-appearance` have vs. not using it here and do we also need [`-moz-appearance`](https://developer.mozilla.org/en/docs/Web/CSS/-moz-appearance#Browser_compatibility) for Firefox? If using just `appearance` on its own, does that get vendor prefixed by our existing autoprefixer infrastructure?